src/main/java/frc/robot/commands/auto/EngageNoPathplanner.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.commands.auto;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.AlignWheelsToZero;
+import frc.robot.commands.BalanceCommand;
+import frc.robot.commands.GoToPose;
+import frc.robot.subsystems.Drivetrain;
+
+public class EngageNoPathplanner extends SequentialCommandGroup{
+
+    private Drivetrain m_drive;
+    private Pose2d rightUnrotatedPose, chargePose, centerPose, rightRotatedPose, leftRotatedPose, leftUnrotatedPose;
+    private Rotation2d rot, startRot;
+
+    /**
+     * Drives out of community, then toward charge station laterally and then drives back to engage. Can be done on either side of drivestation. 
+     * @param drive drivetrain
+     */
+    public EngageNoPathplanner(Drivetrain drive)  {
+        m_drive = drive;
+
+        startRot = new Rotation2d(0);
+        rot = new Rotation2d(Math.PI);
+
+        centerPose = new Pose2d(5.5, 0, startRot);
+        chargePose = new Pose2d(2.54, -1.5, startRot);
+        rightRotatedPose = new Pose2d(5, -1.75, rot);
+        rightUnrotatedPose = new Pose2d(5, -1.75, startRot);
+        leftRotatedPose = new Pose2d(5, 1.75, rot);
+        leftUnrotatedPose = new Pose2d(5, 1.75, startRot);
+
+        addCommands(
+            new AlignWheelsToZero(m_drive),
+            new GoToPose(m_drive, centerPose),
+            new GoToPose(m_drive, rightUnrotatedPose),//alternatively could make it poseRightRotated, but this is untested
+            new GoToPose(m_drive, chargePose),//if starting on left side of charge station, use left pose instead
```

Comment:
make a paramater to switch between left and right or if this is supose to be aliance dependent it a supplier instead. similarly don't use left or right use feild boandry and loading zone.